src/client/datascience/interactive-ipynb/nativeEditorStorage.ts

```diff
@@ -544,6 +561,7 @@ export class NativeEditorStorage implements INotebookStorage {
     public async save(model: INotebookModel, _cancellation: CancellationToken): Promise<void> {
         const contents = model.getContent();
         await this.fileSystem.writeFile(model.file.fsPath, contents, 'utf-8');
+        await this.trustService.updateNotebookTrust(contents, model.isTrusted);
```

Comment:
```suggestion
        await Promise.all([
        	this.fileSystem.writeFile(model.file.fsPath, contents, 'utf-8'),
        	this.trustService.updateNotebookTrust(contents, model.isTrusted)
        ]);
```

I'd do this in parallel, this will happen very frequently hence we need to make this fast.
@rchiodo @joyceerhl do we really need to wait for the trust to be successfully written to disc?

Reply:
Before we can say it's 'saved'? Probably not. You could do these in parallel.

Reply:
Actually doing these in parallel has no effect on waiting for the trust. It still waits. Doing them in parallel just makes it faster. So yeah we should totally do this.

Reply:
Done!